cmd/interactsh-client/main.go

```diff
@@ -86,7 +90,23 @@ func main() {
 		gologger.Fatal().Msgf("Could not parse options: %s\n", err)
 	}
 
-	options.ShowBanner()
+	if cliOptions.PdcpAuth == "true" {
+		options.AuthWithPDCP()
+	} else if len(cliOptions.PdcpAuth) == 36 {
```

Comment:
What does this mean? What do we check here for?

Reply:
if user just do `./interactsh-client -auth`
```console
./interactsh-client -auth

    _       __                       __       __
   (_)___  / /____  _________ ______/ /______/ /_
  / / __ \/ __/ _ \/ ___/ __ '/ ___/ __/ ___/ __ \
 / / / / / /_/  __/ /  / /_/ / /__/ /_(__  ) / / /
/_/_/ /_/\__/\___/_/   \__,_/\___/\__/____/_/ /_/

		projectdiscovery.io

[INF] You are logged in as (@ramana.reddy)
```
it will show their identity.

Reply:
I will add comment on it.